eng/common/instructions/azsdk-tools/sdk-details-in-release-plan.instructions.md

```diff
@@ -5,7 +5,7 @@ description: 'Identify languages configured in the TypeSpec project and add it t
 **Goal**: Identify languages configured in the TypeSpec project and generate the json object with language and package name.
 1. Identify the language emitter configuration in the `tspconfig.yaml` file in the TypeSpec project root.
 2. Identify the package name or namespace for each language emitter.
-   - For Java and Python, use `emitter-output-dir` for package name if it exists. Otherwise use `package-dir` to get the package name as fallback approach.
+   - For Java and Python, use `emitter-output-dir` for package name.
```

Comment:
It is not developed by us so I am not sure whether the change is expected at this time. Let us just keep it unchanged.